linera-core/src/chain_worker/actor.rs

```diff
@@ -431,24 +395,20 @@ where
                 .finish_non_exhaustive(),
             ChainWorkerRequest::ProcessValidatedBlock {
                 certificate,
-                hashed_certificate_values,
```

Comment:
oh right this was only used for bytecode

Reply:
Yes! Glad we're getting to remove this :D